docs/pages/installation/gke.md

```diff
@@ -24,8 +24,8 @@ You may perform the steps outlined under [Managed Kubernetes](#managedk8s), foll
 
 `./generate_kubeconfig_gke.sh cluster-admin-sa-gke default`
 
-Having run this script, fire up the meshery server and head over to the local port, usually found at <code>localhost:3000/settings</code> or <code>localhost:9081/settings</code>. This is where you configure your settings on the adaptor(Istio etc).
-<br>Supply the generated file <code><b> config-cluster-admin-sa-gke-default.yaml </b></code> under kube-config.
+Having run this script, fire up the meshery server and head over to the local port, usually found at `localhost:9081/settings`. This is where you configure your settings on the adaptor(Istio etc).
```

Comment:
Let's strive to use more direct language. The colloquial language is excellent for blog posts, where an individual's voice can be heard. Example:


"Once the script is complete, you may proceed to start Meshery with the GKE-compatible configuration by executing `mesheryctl system start`."

You can remove this sentence: "This is where you configure your settings on the adaptor(Istio etc)."

Reply:
Made the advised changes. Thanks for pointing that out. I'll focus on using more direct language in the future. 
Please review @leecalcote.

Reply:
The last sentence still needs to be removed. The "fire up" phrase is still being used. "head over to" is still being used. They don't need to configure their adapter. They need to run the command I provided to start Meshery.